bin/update_asset

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env ruby
+
+require File.expand_path("../../config/environment", __FILE__)
+
+filename = ARGV.any? ? ARGV.fetch(0) : nil
+old_asset_id = ARGV.any? ? ARGV.fetch(1) : nil
```

Comment:
It would be good to reverse the order of these arguments, to match the [API adapter](https://github.com/alphagov/gds-api-adapters/blob/master/lib/gds_api/asset_manager.rb#L33).